### ISSUE TITLE
ActionBars messed up

### ISSUE THREAD

**Issue by kCenk** [Karl-HeinzSchneider/WoW-DragonflightUI#103]
As you can see after the 0.10 update my Actionbars are messed up, DragonflightUI is the only Addon that i have activated.

Game: Cataclysm Classic

Lua Error:
```
Message: ...e/AddOns/Blizzard_ActionBar/Classic/ActionButton.lua:348: Usage: GetActionTexture(slot)
Time: Thu Jun 20 18:57:14 2024
Count: 1
Stack: ...e/AddOns/Blizzard_ActionBar/Classic/ActionButton.lua:348: Usage: GetActionTexture(slot)
[string "=[C]"]: in function `GetActionTexture'
[string "@Interface/AddOns/Blizzard_ActionBar/Classic/ActionButton.lua"]:348: in function `ActionButton_Update'
[string "@Interface/AddOns/DragonflightUI/Mixin/Actionbar.mixin.lua"]:157: in function `Update'
[string "@Interface/AddOns/DragonflightUI/Mixin/Actionbar.mixin.lua"]:55: in function `SetState'
[string "@Interface/AddOns/DragonflightUI/Modules/Actionbar.lua"]:1706: in function `ApplySettings'
[string "@Interface/AddOns/DragonflightUI/Modules/Actionbar.lua"]:1453: in function <...nterface/AddOns/DragonflightUI/Modules/Actionbar.lua:1439>
[string "=[C]"]: ?
[string "@Interface/AddOns/DragonflightUI/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:66: in function <...ns/DragonflightUI/Libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Interface/AddOns/DragonflightUI/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:523: in function <...ns/DragonflightUI/Libs/AceAddon-3.0/AceAddon-3.0.lua:516>
[string "=(tail call)"]: ?
[string "=(tail call)"]: ?
[string "@Interface/AddOns/DragonflightUI/Modules/Config.lua"]:122: in function `ApplySettings'
[string "@Interface/AddOns/DragonflightUI/Modules/Config.lua"]:91: in function <Interface/AddOns/DragonflightUI/Modules/Config.lua:83>
[string "=[C]"]: ?
[string "@Interface/AddOns/DragonflightUI/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:66: in function <...ns/DragonflightUI/Libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Interface/AddOns/DragonflightUI/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:523: in function `EnableAddon'
[string "@Interface/AddOns/DragonflightUI/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:536: in function `EnableAddon'
[string "@Interface/AddOns/DragonflightUI/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:626: in function <...ns/DragonflightUI/Libs/AceAddon-3.0/AceAddon-3.0.lua:611>

Locals: (*temporary) = nil
```

Screenshot:
![image](https://github.com/Karl-HeinzSchneider/WoW-DragonflightUI/assets/11097109/5b966393-5d59-4284-b33d-70c11036ee2d)

**Comment by Karl-HeinzSchneider** [comment]
I pushed v0.10.1 that should fix (00f827aae9e482a37c26ef4f513362898cbf1c50) this, will be online soon.

**Comment by kCenk** [comment]
Fixed with 10.0.2 didn't get the chance to try 10.0.1 out :) Thanks!
